### ISSUE TITLE
Too much heal in city

### ISSUE THREAD

**Issue by Padre19** [LoneGazebo/Community-Patch-DLL#4426]
_1. Mod version (i.e Date - 4/23):_
5-20-3

_2. Mod list (if using Vox Populi only, leave blank):_
VP + aditional mods ( personal) but this error appears to have occurred after changes have been made to how units fortifying

_3. Error description:_ 
damaged unit, receives too much heal in city (+800 or + 231), instant healed.

_4. Steps to reproduce (optional):_
puts a heavy damaged unit in city and press "Fortify Until Healed".

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by Padre19** [comment]
Sorry. False problem.My mods.....

**Comment by Iamblichos** [comment]
Hmmm....strange can't reproduce. Can you get me a save file with the turn prior to when this happens?

**Comment by Padre19** [comment]
Save file with my mods....Probably from my mods it's this issue...

**Comment by Iamblichos** [comment]
Yeah it doesn’t work that way. The override folder is the database updates.

**Comment by Iamblichos** [comment]
You can’t just add mods to a mod pack. You must create the mod pack with all the mods loaded. I can show you how to do it if you want to know how.

**Comment by Padre19** [comment]
Yes. I know how to create a modpack. But with him I have this error.

**Comment by Padre19** [comment]
Strange....New mod pack.....no problems.
